src/data/dfinsupp/basic.lean

```diff
@@ -20,34 +20,18 @@ universes u u₁ u₂ v v₁ v₂ v₃ w x y l
 
 open_locale big_operators
 
-variables (ι : Type u) {γ : Type w} (β : ι → Type v) {β₁ : ι → Type v₁} {β₂ : ι → Type v₂}
+variables {ι : Type u} {γ : Type w} {β : ι → Type v} {β₁ : ι → Type v₁} {β₂ : ι → Type v₂}
 
-namespace dfinsupp
 
-variable [Π i, has_zero (β i)]
+variable (β)
```

Comment:
What's the point of changing from `{}` to `()` without opening a section/namespace?

Reply:
This was mostly accidental; although I find it's usually clearer to always use `{}` for type variables at the top of the file, as that means they never have to change if things are reordered within the file.